dndwiki/races/race-reindeerfolk.xml

```diff
@@ -0,0 +1,174 @@
+<?xml version="1.0" encoding="utf-8"?>
+<elements>
+	<info>
+		<name>Reindeerfolk</name>
+		<description>The Reindeerfolk Race from the D&amp;D Wiki</description>
+		<author url="https://www.dandwiki.com/wiki/Dessert_Genasi_(5e_Subrace)">D&amp;D Wiki</author>

```

Comment:
Have it link to the current build as DnDwiki changes all the time. You can do this by clicking the date on the latest update: For example, this one should be `https://www.dandwiki.com/w/index.php?title=Dessert_Genasi_(5e_Subrace)&oldid=1625565`